router/router.go

```diff
@@ -255,6 +256,11 @@ func New(cfg *config.Configuration, rateConvertor *currencies.RateConverter) (r
 		glog.Fatalf("Failed to create the video endpoint handler. %v", err)
 	}
 
+	customHeaders := config.CustomHeaders{}
+	if cfg.CustomHeaders != customHeaders {
```

Comment:
This check doesn't scale well if the generic customHeaders struct is later expanded for other purposes. Consider renaming the struct to something more specific such as RequestTimeoutHeaders to discourage future expansion or check for just those two headers specifically.

Reply:
CustomHeaders renamed to RequestTimeoutHeaders